CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 
 ## Improvements
 
-# v1.28.0-rc2 / 2024-07-04
+# v1.28.0-rc3 / 2024-07-04
 
 This is the second release candidate of the upcoming MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
```

Comment:
```suggestion
This is the third release candidate of the upcoming MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
```

Reply:
Let´s also create one line calling out the diff with rc2, so its clearer when people see both rc2 and rc3. Potentially something like:

```
**This release candidate sets the calibration network to upgrade at epoch 1779094, corresponding to 2024-07-11T12:00:00Z.** This release does NOT set the mainnet upgrade epoch yet, in which will be updated in the final release.

Compared to `Lotus v1.28.0-rc2`, the `Lotus v1.28.0-rc3` introduces the manifest for the "control" nodes for F3 passive testing. This change shouldnt have any impact on actual production nodes
```